tests/nodes/NodeConnectionManager.general.test.ts

```diff
@@ -206,6 +207,7 @@ describe(`${NodeConnectionManager.name} general test`, () => {
     await nodeManager.start();
     await nodeConnectionManager.start({
       host: localHost as Host,
+      enableMdns: false,
     });
```

Comment:
i disabled it for tests that didn't need MDNS, because some of those tests would fail expecting some address to be unreachable.

Reply:
Can we make this a bit more robust? So that it works regardless, so less manual configuration. How would I know when mdns should be enabled or not, remember less configuration is better.

Reply:
Ideas? Can we make it just work no matter the situation? If it really is indeed a sort of toggle, it should be `mdns: true/false` I think. But better to just always be enabled, we don't have a situation where it doesn't need it.

Unless dependency injection makes sense here? Thus requiring an external thing to create it first and then inject into it, that would make it a required dependency.

Reply:
i think it would make sense for seednodes to have it disabled, maybe injected dep would make sense

Reply:
this is no longer relevant, due to MDNS being an injected optional dependency